#### Objetivo
Neste caderno vamos utilizar a API do Compras Gvernamentais para identificar fracionamento de compras. O Fracionamento é uma tipologia de irregularidade da licitação que se caracteriza pela realização de compras idênticas no mesmo dia, pelo mesmo órgão, com soma dos valores superior ao limite de dispensa de licitação.

#### Pacotes utilizados

In [39]:
import pandas as pd ### Pacote para análise e transformação de tabelas de dados

### 1. Analisando tabela das licitações de jan a dez/2019 no Portal da Transparência
Fonte: http://portaldatransparencia.gov.br/download-de-dados/licitacoes

#### -> Carregando dados básicos das licitações

In [40]:
df_licitacoes = pd.read_csv('../_dados/compras/201901_Licitaá∆o.csv', sep = ';', encoding = 'latin1')

In [41]:
for i in ['02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']:
    df_temp = pd.read_csv('../_dados/compras/2019' + i + '_Licitaá∆o.csv', sep = ';', encoding = 'latin1')
    df_licitacoes = pd.concat([df_licitacoes, df_temp])

In [42]:
df_licitacoes.head(3)

Número Licitação  Código UG  Código Modalidade Compra  \
0             12017     160501                         2   
1             12017     764100                         7   
2             12018     927248                       -99   

                                        Nome UG             Modalidade Compra  \
0  MUSEU HISTORICO DO EXERCITO FORTE COPACABANA              Tomada de Preços   
1           SERVICO DE IDENTIFICACAO DA MARINHA  Inexigibilidade de Licitação   
2        FUNDO MUNICIPAL DE SAÚDE DE ANANINDEUA    Pregão - Registro de Preço   

     Número Processo                                             Objeto  \
0  64615003280201719  Objeto: Obra de restauração da rede de esgoto ...   
1  63139000949201711   Objeto: Serviço de postagem de correspondências.   
2          8915/2018  Objeto: Pregão Eletrônico -  O objeto da prese...   

               Situação Licitação  Código Órgão Superior  \
0  Evento de Resultado de Julgame                  52000   
1                       Encerrado                  52000   
2  Evento de Retificação Divulgad                  99900   

              Nome Órgão Superior  Código Órgão           Nome Órgão  \
0            Ministério da Defesa         52121  Comando do Exército   
1            Ministério da Defesa         52131   Comando da Marinha   
2  REPUBLICA FEDERATIVA DO BRASIL         93420       ESTADO DO PARA   

        Município Data Resultado Compra Data Abertura Valor Licitação  
0  RIO DE JANEIRO            17/01/2019           NaN     564266,9500  
1  RIO DE JANEIRO            23/01/2019           NaN       5567,2000  
2        Inválido            29/01/2019    30/11/2018      71311,6900

In [43]:
# Filtrando dispensas
df_licitacoes = df_licitacoes[df_licitacoes['Código Modalidade Compra'] == 6]

In [44]:
len(df_licitacoes)

111033

#### -> Carregando dados dos itens

In [45]:
df_itens = pd.read_csv('../_dados/compras/201901_ItemLicitaá∆o.csv', sep = ';', encoding = 'latin1')
for i in ['02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12']:
    df_temp = pd.read_csv('../_dados/compras/2019' + i + '_ItemLicitaá∆o.csv', sep = ';', encoding = 'latin1')
    df_itens = pd.concat([df_itens, df_temp])

In [47]:
df_itens.head(3)

Número Licitação  Código UG  Código Modalidade Compra  \
0             12017     160501                         2   
1             12017     764100                         7   
2             12018     927248                       -99   

                                        Nome UG             Modalidade Compra  \
0  MUSEU HISTORICO DO EXERCITO FORTE COPACABANA              Tomada de Preços   
1           SERVICO DE IDENTIFICACAO DA MARINHA  Inexigibilidade de Licitação   
2        FUNDO MUNICIPAL DE SAÚDE DE ANANINDEUA    Pregão - Registro de Preço   

     Número Processo  Código Órgão           Nome Órgão   CNPJ Vencedor  \
0  64615003280201719         52121  Comando do Exército  06241337000177   
1  63139000949201711         52131   Comando da Marinha  34028316000294   
2          8915/2018         93420       ESTADO DO PARA  63848345000110   

                                 Nome Vencedor  \
0          CRISART CONSTRUCOES E REFORMAS LTDA   
1  EMPRESA BRASILEIRA DE CORREIOS E TELEGRAFOS   
2                             POLYMEDH. EIRELI   

                             Descrição  Quantidade Item   Valor Item  
0  MANUTENCAO DE REDES - AGUA / ESGOTO                1  564266,9500  
1              COMUNICACAO POR CORREIO                1    5567,2000  
2                     'PAPEL BOBINADO'             1620    8748,0000

In [48]:
# Filtrando dispensas
df_itens = df_itens[df_itens['Código Modalidade Compra'] == 6]

In [49]:
len(df_itens)

338804

#### -> Criando tabela conjunta

In [51]:
# Criando id da compra nas tabelas de licitações e itens

def id_compra(x):
    y = str(x['Código UG']) + '06' + '0'*(9-len(str(x['Número Licitação']))) + str(x['Número Licitação'])
    return y

df_licitacoes['id_compra'] = df_licitacoes.apply(id_compra, axis = 1)
df_itens['id_compra'] = df_itens.apply(id_compra, axis = 1)

    
    
    

In [52]:
df_itens_compra = df_itens.merge(df_licitacoes[['id_compra', 'Data Resultado Compra']], how = 'inner', on = 'id_compra')


In [53]:
df_itens_compra.head(3)

Número Licitação  Código UG  Código Modalidade Compra  \
0             12018     926397                         6   
1             12018     365001                         6   
2             12019     926690                         6   

                                         Nome UG      Modalidade Compra  \
0  AGÊN.BRAS.GEST.DE FUNDOS GARANTID.GARANT.S.A.  Dispensa de Licitação   
1             FINANCIADORA DE ESTUDOS E PROJETOS  Dispensa de Licitação   
2          CONSELHO REGIONAL DE ODONTOLOGIA - RJ  Dispensa de Licitação   

  Número Processo  Código Órgão                          Nome Órgão  \
0        004/2018         25000              MINISTERIO DA ECONOMIA   
1          012018         20502  Financiadora de Estudos e Projetos   
2         01/2018         95320            ESTADO DO RIO DE JANEIRO   

    CNPJ Vencedor                        Nome Vencedor  \
0  10242721000161               BUYSOFT DO BRASIL LTDA   
1  04196935000812           GOLDEN DISTRIBUIDORA LTDA.   
2  31302708000120  DIONE BEZERRA MENEZES - INFORMATICA   

                                           Descrição  Quantidade Item  \
0  LICENCIAMENTO DE DIREITOS PERMANENTES  DE  USO...                2   
1                                     MONITOR IMAGEM                1   
2        CARTUCHO TONER IMPRESSORA / COPIADORA XEROX                2   

  Valor Item          id_compra Data Resultado Compra  
0  6200,0000  92639706000012018            02/01/2019  
1  2173,0800  36500106000012018            10/01/2019  
2   300,0000  92669006000012019            09/01/2019

In [54]:
len(df_itens_compra)

338804

In [55]:
list(df_itens_compra)

['Número Licitação',
 'Código UG',
 'Código Modalidade Compra',
 'Nome UG',
 'Modalidade Compra',
 'Número Processo',
 'Código Órgão',
 'Nome Órgão',
 'CNPJ Vencedor',
 'Nome Vencedor',
 'Descrição',
 'Quantidade Item',
 'Valor Item',
 'id_compra',
 'Data Resultado Compra']

In [56]:
df_itens_compra['Valor Item'] = df_itens_compra['Valor Item'].apply(lambda x: float(x.replace(',', '.')))

In [57]:
df_itens_compra.to_pickle('df_itens_compra.pkl')

In [58]:
df_itens_compra = pd.read_pickle('df_itens_compra.pkl')

#### -> Identificando potenciais fracionamentos

In [60]:
df_itens_compra = df_itens_compra[df_itens_compra['Valor Item'] < 17600]


In [62]:
df_itens_compra_agregado = df_itens_compra[['id_compra', 'Data Resultado Compra', 'Código UG', 'Nome UG', 'Descrição', 'Valor Item']].groupby(['Data Resultado Compra', 'Código UG', 'Nome UG', 'Descrição']).agg({'Valor Item': 'sum', 'id_compra': 'count'}).reset_index()
df_itens_compra_agregado.columns = ['Data Resultado Compra', 'Código UG', 'Nome UG', 'Descrição', 'soma_valor', 'num_itens']


In [63]:
df_indicio_fracionamento = df_itens_compra_agregado[(df_itens_compra_agregado['soma_valor'] > 17600) & (df_itens_compra_agregado['num_itens'] > 1)]


In [64]:
df_indicio_fracionamento.head(3)

Data Resultado Compra  Código UG                               Nome UG  \
18            01/02/2019     114608  UNIDADE ESTADUAL DO IBGE NO MARANHãO   
74            01/02/2019     155009                        EBSERH HUB-UNB   
83            01/02/2019     155021                        EBSERH HU-UFMG   

                                            Descrição  soma_valor  num_itens  
18  PRESTACAO DE SERVICO DE LIMPEZA E CONSERVACAO ...     41272.0          5  
74              MANUTENCAO GERADOR - VAPOR / GASOLINA     28100.0          6  
83                                          ANTICORPO     42133.0          4

In [65]:
len(df_indicio_fracionamento)

1587

Já conseguimos aqui chegar a 1587 indícios de fracionamento. Mas só com os dados do Portal não é possível se ter certeza, pois lá não é informado o inciso legal que justificou a dispensa. Lembrando que o fracionamento ocorre quando o motivo é o valor ser inferior ao limite que autoriza a compra direta, mas a ocorrência de outras compras idênticas fazem com que a soma dos valores supere esse limite.

Por isso, na próxima etapa vamos obter os incisos de justificativa lá na API do Compras Governamentais. Um questionamento válido seria "Por que não pegar logo essa informação para todas as compras, desde o incício?". O objetivo de chegarmos num grupo menor de indícios primeiro é economizar na utilização da API de compras, que não é muito rápida nas respostas das requisições.

### 2. Trazendo informação do inciso de justificativa da API do Compras Governamentais


#### -> Criando lista de id_compra de potenciais fracionamentos

In [75]:
df_itens_compra_ocorrencias = df_itens_compra.merge(df_indicio_fracionamento[['Data Resultado Compra', 'Código UG', 'Descrição']], how = 'inner', on = ['Data Resultado Compra', 'Código UG', 'Descrição'])


In [76]:
len(df_itens_compra_ocorrencias)

13805

In [77]:
lista_id_compra_ocorrencias = list(set(df_itens_compra_ocorrencias['id_compra']))

In [78]:
len(lista_id_compra_ocorrencias)

2405

#### -> Conectando API e trazendo informação do inciso
Não exige credencial. Só acessar o módulo desejado em: http://compras.dados.gov.br/docs/home.html

No nosso caso, queremos investigar as compras sem licitação: http://compras.dados.gov.br/docs/compraSemLicitacao/v1/compras_slicitacao.html

In [ ]:
raiz_url = 'http://compras.dados.gov.br/compraSemLicitacao/id/compra_slicitacao/'
id_compra = lista_id_compra_ocorrencias[0]
url = raiz_url + id_compra + '.csv'
df_compras_api = pd.read_csv(url)
df_compras_api['id_compra'] = id_compra

In [ ]:
count = 1

for i in lista_id_compra_ocorrencias[1:]:
    id_compra = i
    url = raiz_url + id_compra + '.csv'
    df_temp = pd.read_csv(url)
    df_temp['id_compra'] = id_compra
    df_compras_api = pd.concat([df_compras_api, df_temp])
    count+=1
    print(count, len(df_compras_api))
    

In [ ]:
len(df_compras_api)

In [ ]:
df_compras_api.to_pickle('df_compras_api.pkl')

In [ ]:
df_compras_api.head(3)

#### -> Agora, vamos juntar essa informação na nossa tabela de indícios

In [79]:
df_compras_api = pd.read_pickle('df_compras_api.pkl')

In [80]:
df_itens_compra_ocorrencias = df_itens_compra_ocorrencias.merge(df_compras_api[['id_compra', 'Número Inciso']], how = 'inner', on = 'id_compra')


### 3. Agora vamos refazer a lógica de identificação do fracionamento, mas somente para as potenciais ocorrências em que o inciso de justificativa foi o 2 (valor menor que limite).

In [81]:
df_itens_compra_ocorrencias = df_itens_compra_ocorrencias[df_itens_compra_ocorrencias['Número Inciso'] == 2]

In [82]:
len(df_itens_compra_ocorrencias)

2877

#### -> Agregando os dados para chegar aos fracionamentos

In [83]:
df_fracionamento = df_itens_compra_ocorrencias[['id_compra', 'Data Resultado Compra', 'Código UG', 'Nome UG', 'Descrição', 'Valor Item']].groupby(['Data Resultado Compra', 'Código UG', 'Nome UG', 'Descrição']).agg({'Valor Item': 'sum', 'id_compra': 'count'}).reset_index()
df_fracionamento.columns = ['Data Resultado Compra', 'Código UG', 'Nome UG', 'Descrição', 'soma_valor', 'num_itens']


In [84]:
df_fracionamento

Data Resultado Compra  Código UG  \
0              01/02/2019     114608   
1              01/02/2019     155009   
2              01/02/2019     155021   
3              01/03/2019     158289   
4              01/07/2019     240120   
5              01/08/2019     753000   
6              01/10/2019     254445   
7              01/11/2019     153063   
8              01/11/2019     323028   
9              01/11/2019     785800   
10             02/04/2019     155021   
11             02/05/2019     153490   
12             02/05/2019     154040   
13             02/05/2019     791610   
14             02/07/2019     413007   
15             02/07/2019     926137   
16             02/08/2019     135012   
17             02/08/2019     155021   
18             02/08/2019     160193   
19             02/08/2019     170201   
20             02/09/2019     160176   
21             02/12/2019     135015   
22             02/12/2019     160234   
23             02/12/2019     160267   
24             03/01/2019     170321   
25             03/01/2019     926397   
26             03/04/2019     110096   
27             03/04/2019     135007   
28             03/05/2019     155009   
29             03/05/2019     403201   
..                    ...        ...   
553            30/12/2019     135006   
554            30/12/2019     153063   
555            30/12/2019     153063   
556            30/12/2019     170139   
557            30/12/2019     240125   
558            30/12/2019     370003   
559            31/01/2019     240120   
560            31/07/2019     155023   
561            31/07/2019     155905   
562            31/07/2019     160196   
563            31/07/2019     160536   
564            31/07/2019     344042   
565            31/10/2019     135014   
566            31/10/2019     153063   
567            31/10/2019     153063   
568            31/10/2019     153063   
569            31/10/2019     153063   
570            31/10/2019     153063   
571            31/10/2019     153063   
572            31/10/2019     153063   
573            31/10/2019     153063   
574            31/10/2019     153063   
575            31/10/2019     153063   
576            31/10/2019     153063   
577            31/10/2019     250061   
578            31/12/2019     113601   
579            31/12/2019     135016   
580            31/12/2019     153173   
581            31/12/2019     170135   
582            31/12/2019     194045   

                                           Nome UG  \
0             UNIDADE ESTADUAL DO IBGE NO MARANHãO   
1                                   EBSERH HUB-UNB   
2                                   EBSERH HU-UFMG   
3              INST.FED.DO MARANHAO/CAMPUS ZE DOCA   
4           CENTRO BRASILEIRO DE PESQUISAS FISICAS   
5      INSTITUTO DE EST. DO MAR ALTE PAULO MOREIRA   
6       INSTITUTO DE TECNOLOGIA EM IMUNOBIOLOGICOS   
7                     UNIVERSIDADE FEDERAL DO PARA   
8             AGENCIA NACIONAL DE ENERGIA ELETRICA   
9                      ESTACAO NAVAL DO RIO GRANDE   
10                                  EBSERH HU-UFMG   
11                      FACULDADE DE LETRAS - UFRJ   
12         FUNDACAO UNIVERSIDADE DE BRASILIA - FUB   
13       COMANDO DO PRIMEIRO ESQUADRAO DE ESCOLTAS   
14         AGENCIA NACIONAL DE TELECOMUNICACOES-PE   
15    GER. DE CENTRO DE SER.COMP.ESCRITÓRIOCENTRAL   
16                                  EMBRAPA/CPATSA   
17                                  EBSERH HU-UFMG   
18     COMANDO 10 BRIGADA DE INFANTARIA MOTORIZADA   
19                 DELEGACIA DA RFB EM PORTO VELHO   
20             COMANDO 1º GRUPAMENTO DE ENGENHARIA   
21                                   EMBRAPA/CNPGL   
22                5 REGIMENTO DE CARROS DE COMBATE   
23        2 BATALHAO DE INFANTARIA MOTORIZADO (ES)   
24                  DELEGACIA DA RFB EM PIRACICABA   
25   AGÊN.BRAS.GEST.DE FUNDOS GARANTID.GARANT.S.A.   
26    SUPERINTENDENCIA ADMINISTRACAO EM PERNAMBUCO   
27         

In [85]:
df_fracionamento = df_fracionamento[(df_fracionamento['soma_valor'] > 17600) & (df_fracionamento['num_itens'] > 1)].reset_index(drop = 1)


In [86]:
df_fracionamento

Data Resultado Compra  Código UG  \
0              01/02/2019     114608   
1              01/02/2019     155009   
2              01/02/2019     155021   
3              01/07/2019     240120   
4              01/08/2019     753000   
5              01/10/2019     254445   
6              01/11/2019     153063   
7              01/11/2019     323028   
8              01/11/2019     785800   
9              02/04/2019     155021   
10             02/05/2019     153490   
11             02/05/2019     154040   
12             02/05/2019     791610   
13             02/07/2019     413007   
14             02/07/2019     926137   
15             02/08/2019     135012   
16             02/08/2019     155021   
17             02/08/2019     160193   
18             02/08/2019     170201   
19             02/09/2019     160176   
20             02/12/2019     135015   
21             02/12/2019     160234   
22             02/12/2019     160267   
23             03/01/2019     170321   
24             03/01/2019     926397   
25             03/04/2019     110096   
26             03/04/2019     135007   
27             03/05/2019     155009   
28             03/05/2019     403201   
29             03/06/2019     113202   
..                    ...        ...   
526            30/10/2019     250031   
527            30/12/2019     113601   
528            30/12/2019     135006   
529            30/12/2019     153063   
530            30/12/2019     153063   
531            30/12/2019     170139   
532            30/12/2019     370003   
533            31/01/2019     240120   
534            31/07/2019     155023   
535            31/07/2019     155905   
536            31/07/2019     160196   
537            31/07/2019     160536   
538            31/10/2019     135014   
539            31/10/2019     153063   
540            31/10/2019     153063   
541            31/10/2019     153063   
542            31/10/2019     153063   
543            31/10/2019     153063   
544            31/10/2019     153063   
545            31/10/2019     153063   
546            31/10/2019     153063   
547            31/10/2019     153063   
548            31/10/2019     153063   
549            31/10/2019     153063   
550            31/10/2019     250061   
551            31/12/2019     113601   
552            31/12/2019     135016   
553            31/12/2019     153173   
554            31/12/2019     170135   
555            31/12/2019     194045   

                                           Nome UG  \
0             UNIDADE ESTADUAL DO IBGE NO MARANHãO   
1                                   EBSERH HUB-UNB   
2                                   EBSERH HU-UFMG   
3           CENTRO BRASILEIRO DE PESQUISAS FISICAS   
4      INSTITUTO DE EST. DO MAR ALTE PAULO MOREIRA   
5       INSTITUTO DE TECNOLOGIA EM IMUNOBIOLOGICOS   
6                     UNIVERSIDADE FEDERAL DO PARA   
7             AGENCIA NACIONAL DE ENERGIA ELETRICA   
8                      ESTACAO NAVAL DO RIO GRANDE   
9                                   EBSERH HU-UFMG   
10                      FACULDADE DE LETRAS - UFRJ   
11         FUNDACAO UNIVERSIDADE DE BRASILIA - FUB   
12       COMANDO DO PRIMEIRO ESQUADRAO DE ESCOLTAS   
13         AGENCIA NACIONAL DE TELECOMUNICACOES-PE   
14    GER. DE CENTRO DE SER.COMP.ESCRITÓRIOCENTRAL   
15                                  EMBRAPA/CPATSA   
16                                  EBSERH HU-UFMG   
17     COMANDO 10 BRIGADA DE INFANTARIA MOTORIZADA   
18                 DELEGACIA DA RFB EM PORTO VELHO   
19             COMANDO 1º GRUPAMENTO DE ENGENHARIA   
20                                   EMBRAPA/CNPGL   
21                5 REGIMENTO DE CARROS DE COMBATE   
22        2 BATALHAO DE INFANTARIA MOTORIZADO (ES)   
23                  DELEGACIA DA RFB EM PIRACICABA   
24   AGÊN.BRAS.GEST.DE FUNDOS GARANTID.GARANT.S.A.   
25    SUPERINTENDENCIA ADMINISTRACAO EM PERNAMBUCO   
26                                  EMBRAPA/CNPASA   
27         

In [87]:
df_fracionamento.to_pickle('df_fracionamento.pkl')

É isso. Chegamos aqui a 556 grupos de aquisições (cada linha é 1 grupo) que compõem possíveis fracionamentos. São compras do mesmo material, no mesmo dia, pela mesma unidade gestora, justificadas (cada aquisição direta) pelo valor menor que R$ 17600, mas cuja soma dos valores supera esse limite. Então, o que se imagina é que o gestor tinha uma compra maior e fatiou-a para permitir a aquisição direta. Vale lembrar que são indícios. Para se confirmar é necessário uma análise mais pormenorizada dos processos de compra. Um possível fator que gera falso-positivo nesta análise é a descrição genérica do material comprado (deixei de propósito alguns exemplos acima, como "informação protegida por sigilo").

#### -> Visualizando detalhes das ocorrências

In [88]:
# Função para obter as informações detalhadas das compras de cada ocorrência

def detalha_ocorrencia(x):
    data = df_fracionamento.iloc[x]['Data Resultado Compra']
    ug = df_fracionamento.iloc[x]['Código UG']
    material = df_fracionamento.iloc[x]['Descrição']
    y = df_itens_compra_ocorrencias[(df_itens_compra_ocorrencias['Data Resultado Compra'] == data) &\
                                    (df_itens_compra_ocorrencias['Código UG'] == ug) &\
                                    (df_itens_compra_ocorrencias['Descrição'] == material)]
    return y

In [90]:
# Escolha um índice da tabela df_fracionamento para obter os detalhes:
detalha_ocorrencia(1)

Número Licitação  Código UG  Código Modalidade Compra         Nome UG  \
1051         111042019     155009                         6  EBSERH HUB-UNB   
1052         111042019     155009                         6  EBSERH HUB-UNB   
1053         111042019     155009                         6  EBSERH HUB-UNB   
1054         111042019     155009                         6  EBSERH HUB-UNB   
1055         111042019     155009                         6  EBSERH HUB-UNB   
1056         111042019     155009                         6  EBSERH HUB-UNB   

          Modalidade Compra    Número Processo  Código Órgão  \
1051  Dispensa de Licitação  23522011104201893         26443   
1052  Dispensa de Licitação  23522011104201893         26443   
1053  Dispensa de Licitação  23522011104201893         26443   
1054  Dispensa de Licitação  23522011104201893         26443   
1055  Dispensa de Licitação  23522011104201893         26443   
1056  Dispensa de Licitação  23522011104201893         26443   

                                       Nome Órgão   CNPJ Vencedor  \
1051  Empresa Brasileira de Serviços Hospitalares  72588080000159   
1052  Empresa Brasileira de Serviços Hospitalares  72588080000159   
1053  Empresa Brasileira de Serviços Hospitalares  72588080000159   
1054  Empresa Brasileira de Serviços Hospitalares  72588080000159   
1055  Empresa Brasileira de Serviços Hospitalares  72588080000159   
1056  Empresa Brasileira de Serviços Hospitalares  72588080000159   

                                   Nome Vencedor  \
1051  POWER ENGENHARIA COMERCIO DE SERVICOS LTDA   
1052  POWER ENGENHARIA COMERCIO DE SERVICOS LTDA   
1053  POWER ENGENHARIA COMERCIO DE SERVICOS LTDA   
1054  POWER ENGENHARIA COMERCIO DE SERVICOS LTDA   
1055  POWER ENGENHARIA COMERCIO DE SERVICOS LTDA   
1056  POWER ENGENHARIA COMERCIO DE SERVICOS LTDA   

                                  Descrição  Quantidade Item  Valor Item  \
1051  MANUTENCAO GERADOR - VAPOR / GASOLINA                1      9800.0   
1052  MANUTENCAO GERADOR - VAPOR / GASOLINA                1      2700.0   
1053  MANUTENCAO GERADOR - VAPOR / GASOLINA                1      2500.0   
1054  MANUTENCAO GERADOR - VAPOR / GASOLINA                1      3500.0   
1055  MANUTENCAO GERADOR - VAPOR / GASOLINA                1      4800.0   
1056  MANUTENCAO GERADOR - VAPOR / GASOLINA                1      4800.0   

              id_compra Data Resultado Compra  Número Inciso  
1051  15500906111042019            01/02/2019              2  
1052  15500906111042019            01/02/2019              2  
1053  15500906111042019            01/02/2019              2  
1054  15500906111042019            01/02/2019              2  
1055  15500906111042019            01/02/2019              2  
1056  15500906111042019            01/02/2019              2